piecrust/tests/initializer.rs

```diff
@@ -40,12 +40,6 @@ fn init() -> Result<(), Error> {
         result.is_err(),
         "calling init directly as transaction should not be allowed"
     );
-    // we should not be able to call init as query neither
-    let result = session.call::<u8, ()>(id, CONTRACT_INIT_METHOD, &0xaa, LIMIT);
-    assert!(
-        result.is_err(),
-        "calling init directly as query should not be allowed"
-    );
```

Comment:
Why this has been removed?

Reply:
Because it was duplicated - exactly the same call twice, probably because in the past queries differed from calls and these tests were different but after query removal they became the same